README.rst

```diff
@@ -43,7 +43,6 @@ Any good internet browser will be able to display the documentation.
 
 To build the pdf form of the documentation simply execute the following command::
 
-  make pdf
-
-
+  make latexpdf
 
+Note: Creating the pdf form of the documentation requires that latexmk is available.
```

Comment:
`PDF` not `pdf`. As for `latexmk`, we should mention where to get it from. Clearly, neither from `pip[3]` nor `brew`.

Reply:
It does come from `brew` just not as easy as `brew install latexmk`.

Reply:
A `brew search latexmk` yields:

```
No formula or cask found for "latexmk".
Closed pull requests:
New package: Latexmk (4.31) (https://github.com/Homebrew/legacy-homebrew/pull/13580)
Latexmk 4.31 (new package) (https://github.com/Homebrew/legacy-homebrew/pull/13578)
```

So, not available from `brew` anymore, it would seem. Have to install it through MacTex (see https://mg.readthedocs.io/latexmk.html).